core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -445,8 +445,8 @@ class MapUnit {
         if (closestCity == null) return
         val distance = closestCity.getCenterTile().aerialDistanceTo(tile)
         var productionPointsToAdd = if (distance == 1) 20 else 20 - (distance - 2) * 5
-        if (tile.owningCity == null || tile.owningCity!!.civInfo != civInfo) productionPointsToAdd = productionPointsToAdd * 2 / 3
```

Comment:
I understand the purpose, but removing the if and adding `if (productionPointsToAdd == 0) return` is even better :)

Reply:
Dang shouldn't have touched this... This function is also another place where Studio cries for a `?:` or `@Suppress("FoldInitializerAndIfToElvis")` - and I didn't touch _that_!!!